seatreg.php

```diff
@@ -86,4 +88,29 @@ function seatreg_plugin_deactivate() {
 }
 
 //public API
-require( 'php/public_api.php' );
\ No newline at end of file
+require( 'php/public_api.php' );
+
+//Capabilities
+function seatreg_capabilities_add() {
+	$role = get_role('administrator');
+	if ( !$role->has_cap('seatreg_manage_events') ) {
+		$role->add_cap('seatreg_manage_events');
+	}
+	if ( !$role->has_cap('seatreg_manage_bookings') ) {
+		$role->add_cap('seatreg_manage_bookings');
+	}
+}
+function seatreg_capabilities_remove() {
+	$role = get_role('administrator');
+	if ( $role->has_cap('seatreg_manage_events') ) {
+		$role->remove_cap('seatreg_manage_events');
+	}
+	if ( $role->has_cap('seatreg_manage_bookings') ) {
+		$role->remove_cap('seatreg_manage_bookings');
+	}
+}
```

Comment:
What would you say if we relocate those functions to seatreg_functions.php file?